### PR TITLE
TOMEE-2583 - add translate in PT to examples/datasource-versioning

### PR DESCRIPTION
**Pull request by robinsonvs**
ticket of Jira = https://issues.apache.org/jira/browse/TOMEE-2583

### CODE REVIEW

File: examples/datasource-versioning/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+:jbake-status: status=published
+= Versionamento de DataSource
+
+Este exemplo mostra como usar DataSources com versão do mesmo provedor usando o atributo classpath.
+
+== Configuração
+
+A configuração do DataSource pode ser feita de várias maneiras e aqui nós mostramos
+dois métodos comuns na forma de testes unitários. Antes de começarmos, se você
+der uma olhada no pom.xml e procurar por maven-dependencia-plugin em uso, você verá que nós puxamos em dois
```
Comment (marcoantoniobferreira) [inline examples/datasource-versioning/README_pt.adoc]: In this sentence "e procurar por maven-dependencia-plugin em uso", i think that makes more sense like that "e procurar pelo uso do maven-depedency-plugin"

Comment (robinsonvs) [reply examples/datasource-versioning/README_pt.adoc]: Hi Marco, I fix the mistake, thanks

Comment (robinsonvs) [reply examples/datasource-versioning/README_pt.adoc]: I comment on commit, sorry, I'm nervous, it's my first open source pull request

Comment (marcoantoniobferreira) [reply examples/datasource-versioning/README_pt.adoc]: No problem my friend =)

**Comment by cesarhernandezgt** [comment]
Nice job @robinsonvs

**Pull request merged**